docs/config.rst

```diff
@@ -111,6 +111,12 @@ Core config section
     The original MPD server only supports 10000 tracks in the tracklist. Some
     MPD clients will crash if this limit is exceeded.
 
+.. confval:: core/restore_state
+
+    When set to ``true``, Mopidy saves the state when it ends and 
+    restores the state at next start.
```

Comment:
I think "state" should be explained a bit more here, as this is the primary spot for end users to learn about the feature.

"When set to `true`, Mopidy restores its last state when started. The restored state includes the tracklist, playback history, the playback state, and the mixer volume."